stix_shifter_modules/guardium/stix_translation/query_constructor.py

```diff
@@ -2,23 +2,44 @@
 import json
 import datetime
 import copy
-from stix_shifter_utils.stix_translation.src.patterns.pattern_objects import ObservationExpression, ComparisonExpression, \
+from stix_shifter_utils.stix_translation.src.patterns.pattern_objects import ObservationExpression, \
+    ComparisonExpression, \
     ComparisonExpressionOperators, ComparisonComparators, Pattern, \
-    CombinedComparisonExpression, CombinedObservationExpression, ObservationOperators
+    CombinedComparisonExpression, CombinedObservationExpression, ObservationOperators, StartStopQualifier, SetValue
 from stix_shifter_modules.guardium.stix_translation.transformers import TimestampToGuardium
 from stix_shifter_utils.stix_translation.src.json_to_stix import observable
 from stix_shifter_utils.utils.file_helper import read_json
 from stix_shifter_utils.utils import logger
 
-
 DEFAULT_DAYS_BACK = 2
 
 
 class QueryStringPatternTranslator:
+    # Change comparator values to match with supported data source operators
```

Comment:
remove the comparator_lookup, it's not needed since they are now defined in operators.json.